src/transforms/operationTransforms.ts

```diff
@@ -291,13 +292,22 @@ export function transformOperationResponse(
 
   const isDefault = httpInfo.statusCodes.indexOf("default") > -1;
 
-  return {
-    statusCodes: httpInfo.statusCodes,
```

Comment:
We weren't relying on the statusCodes for determining which error to use, were we? (E.g. if an operation supports multiple error types based on the status code)

Reply:
Nope, currently we only handle Errors marked by `x-ms-error-response` or `default` status code. When a response is marked as `x-ms-error-response` M4 groups it under exceptions instead of Error. If a swagger defines an error response without that extension it will be treated as a success response and we'll very likely throw. However that would be a buggy swagger.